src/swe/swe_ceed.c

```diff
@@ -3,6 +3,14 @@
 
 #include "swe_ceed_impl.h"
 
+// CEED uses C99 VLA features for shaping multidimensional
+// arrays, which don't have the same drawbacks as VLA allocations.
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wvla"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wvla"
```

Comment:
If one complies with an intel compiler, would we need to add a new pragma?

Reply:
Btw, we are presently using gcc on Perlmutter and Frontier.

Reply:
And this branch compiles just fine on both Perlmutter and Frontier.

Reply:
These pragmas disable the warnings that we've put in place for GCC and Clang. I'm not sure if Intel even has a warning option for VLAs, but if we don't enable such a thing, we don't need the pragmas here.

GCC and Clang warnings should be enough to keep us from using VLAs unintentionally, because our CI system probably uses GCC in all cases.